src/store/conversationsStore.js

```diff
@@ -26,6 +26,7 @@ import {
 	setSIPEnabled,
 	changeLobbyState,
 	changeReadOnlyState,
+	changeConversationState,
```

Comment:
I suggest you simply reuse changeReadOnlyState and extend it with the new attribute values where needed, no need for an extra method.

This means you'll also be able to reuse this.conversation.readOnly to contain the new state as well